modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/org.elasticsearch.txt

```diff
@@ -177,5 +177,6 @@ class org.elasticsearch.painless.FeatureTest no_import {
 
 # for testing
 static {
+  float staticAddFloatsTest(float, float) from org.elasticsearch.painless.FeatureTest
```

Comment:
(For a followup): can we move these test specific cases out of the base whitelist, since we have whitelist spi (I know we didn't have that when FeatureTest was added above)?

Reply:
@rjernst Yes, in fact given that we can do custom whitelists now, is there any reason not to move FeatureTest and BindingTest to the test framework and use a custom whitelist with those during the unit tests?

Reply:
I don't think they need to be part of src, or in the base whitelist. They can be exposed to tests through the compiler/contexts created in ScriptTestCase.